tutorials/vecops/vo002_VectorCalculations.C

```diff
@@ -16,9 +16,9 @@ void vo002_VectorCalculations()
 {
 
    // Operations on RVec instances are made to be *fast* (vectorisation is exploited)
-   // and easy to use.
-   RVec<float> v1{1., 2., 3.};
-   RVec<float> v2{4., 5., 6.};
+   // and easy to use. RVecF, RVecD, RVecI, ..., are handy aliases for RVec<float>, RVec<double>, RVec<int>, ...
+   ROOT::RVecF v1{1., 2., 3.};
```

Comment:
Should we adjust the above `using namespace ROOT::VecOps;` to be able to spell `RVecF` instead of `ROOT::RVecF`?

Reply:
I had thought about it and in the end personally I like `ROOT::RVecF` more (and `using namespace ROOT::VecOps` is still useful to be able to write e.g. `Mean(v)`). I realize it's quite subjective

Reply:
Ah wait, ADL is a thing! xD let me see if we can do without the `using namespace ROOT::VecOps`...

Reply:
done